storage/simulation/cluster.go

```diff
@@ -57,11 +57,12 @@ type Cluster struct {
 	epoch           int
 	epochWriter     *tabwriter.Writer
 	actionWriter    *tabwriter.Writer
+	script          *Script
```

Comment:
no need for this to be a pointer. i can see it's convenient because that's what the constructor returns, but just deref it before passing to the constructor